lib/private/Files/Filesystem.php

```
@@ -319,11 +314,7 @@ public static function getMountByNumericId($id) {
 	 */
 	public static function resolvePath($path) {
 		$mount = self::getMountManager()->find($path);
-		if ($mount) {
-			return [$mount->getStorage(), rtrim($mount->getInternalPath($path), '/')];
-		} else {
-			return [null, null];
-		}
+		return [$mount->getStorage(), rtrim($mount->getInternalPath($path), '/')];
```

I guess we now always get a mount regardless, and instead of null it would be the Root mount ?

---

Yes, it now throws an exception instead of returning a `null` when no mount is found as "that should never happen" and it wasn't handled consistently anyway.